common/app/common/NewNavigation.scala

```diff
@@ -75,7 +75,7 @@ object NewNavigation {
 
     val uk = NavLinkLists(
       List(headlines, ukNews, world, business, environment, tech, politics),
-      List(science, globalDevelopment, cities, obituaries)
+      List(science, cities, obituaries, globalDevelopment)
```

Comment:
You may want to take a look at other editions. AU for example has some long ones.